recipe/meta.yaml

```diff
@@ -1,4 +1,5 @@
 {% set name = "osmnx" %}
+{% set base = "osmnx-base" %}
 {% set version = "2.0.0" %}
 {% set python_min = "3.9" %}
```

Comment:
We only need to define this if the min python is higher then the EOL. Python 3.9 is conda-forge's min now.

```suggestion
```